src/components/SQForm/SQFormMaskedTextField.tsx

```diff
@@ -1,14 +1,52 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import {TextFieldProps, InputProps} from '@material-ui/core';
 import MaskedInput from 'react-text-mask';
+import BaseFieldProps from 'types/BaseFieldProps';
+import {maskProp} from '../../types/MaskTypes';
 import SQFormTextField from './SQFormTextField';
 
-function TextFieldMask({inputRef, mask, ...rest}) {
+interface SQFormMaskedTextFieldProps extends BaseFieldProps {
+  /** Valid mask array; custom or from utils/masks.js */
+  mask?: maskProp;
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder?: string;
+  /** Disabled property to disable the input if true */
+  isDisabled?: boolean;
+  /** Required property used to highlight input and label if not fulfilled */
+  isRequired?: boolean;
+  /** Custom onBlur event callback */
+  onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
+  /** Custom onChange event callback */
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Adornment that appears at the start of the input */
+  startAdornment?: React.ReactElement;
+  /** Adornment that appears at the end of the input */
+  endAdornment?: React.ReactElement;
+  /** Defines the input type for the text field. Must be a valid HTML5 input type */
+  type?: string;
+  /** Props applied to the Input element */
+  InputProps?: InputProps;
+  /** Attributes applied to the `input` element */
+  inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
+  /** Any valid prop for material ui text input child component - https://material-ui.com/api/text-field/#props */
+  muiFieldProps?: TextFieldProps;
+}
```

Comment:
Since the props for this component are identical to SQFormTextField, other than `mask`, we could just export the `SQFormTextFieldProps` interface and use it here.

```suggestion
import { SQFormTextFieldProps } from './SQFormTextField';

interface SQFormMaskedTextFieldProps extends SQFormTextFieldProps {
  /** Valid mask array; custom or from utils/masks.js */
  mask?: maskProp;
}
```